### DISCUSSION TITLE
Please release new version

### DISCUSSION THREAD

**Discussion by qarmin** [plotters-rs/plotters#577] [General]
Last version is almost 1 year old, and quite a few useful features have been added since then

**Comment by jayvdb** [comment]
There are also considerable problems with https://github.com/plotters-rs/plotters/releases/tag/v0.3.5 , because it uses old dependencies, esp font-kit

- https://github.com/plotters-rs/plotters/issues/548
- https://github.com/plotters-rs/plotters/issues/564
- https://github.com/plotters-rs/plotters/issues/569
- https://github.com/plotters-rs/plotters/issues/573

**Comment by AaronErhardt** [comment]
Done :)

**Comment by pbdeuchler** [comment]
I know it's annoying but any chance we could get another release now that https://github.com/plotters-rs/plotters/pull/607 has been merged?

**Comment by pbdeuchler** [comment]
Bump @AaronErhardt

**Reply by AaronErhardt** [reply]
Done

**Reply by pbdeuchler** [reply]
Immensely appreciated, thank you!
